app/migrations/seed.py

```diff
@@ -1,182 +1,125 @@
-"""
-TODO(ussd): Migration code from GE - need to translate to Python
+import sys
+import os
 
-CommunityToken.where(name: 'Gatina').first_or_create! do |c|
-  c.name = 'Gatina'
-  c.token_alias = 'GATINA'
-  c.token_identifier = '5b9ccc1a81df57e455340a3f'
-  c.contract_address = '0x826a4205b829d1907930b1632f9ce227072e59ca'
-end
+sys.path.append(os.path.abspath(os.path.join(os.getcwd(), "..", "..")))
+sys.path.append(os.path.abspath(os.path.join(os.getcwd(), "..")))
 
-CommunityToken.where(name: 'Bangla').first_or_create! do |c|
-  c.name = 'Bangla'
-  c.token_alias = 'BANGLA'
-  c.token_identifier = '5b9ccb1c81df57e4553402e8'
-  c.contract_address = '0x59c7eec63b7ccd438a6dc428848973f11d894b7d'
-end
+from server import db, create_app
+from server.models.ussd import UssdMenu
 
-CommunityToken.where(name: 'Lindi').first_or_create! do |c|
-  c.name = 'Lindi'
-  c.token_alias = 'LINDI'
-  c.token_identifier = '5b9ccdbb81df57e455341650'
-  c.contract_address = '0x8a890d89cd3585b06a48a174dd847d7818d3074f'
-end
 
-CommunityToken.where(name: 'Kangemi').first_or_create! do |c|
-  c.name = 'Kangemi'
-  c.token_alias = 'KANG'
-  c.token_identifier = '5b9cccbc81df57e455340ee8'
-  c.contract_address = '0x22e9839aeff9818a8b3617586167e8b92f2ed3c4'
-end
+def get_or_create_menu(name, **kwargs):
```

Comment:
wondering why ur setting kwargs rather than explicitly specifying the arguments allowed? 
Given it's a factory for a single class, we should be able to list them all, which would help prevent incorrect args creeping in no?

Reply:
good idea. fixed.
for another PR -- can we also add a seed for creating a user and a token? i dropped my database to test this and lost my user and token that we created that other time through the test python script. would be good if we populated our local dbs with enough stuff to be able to run from the get go

Reply:
i guess that's a bit different actually. there's a seed that we need to run everywhere -- which is the case for this script since we need the menus on prod as well. and then there's one we need to run locally to have a good dev setup like user and token

Reply:
I'll just create different triggers for them, but it make sense to have them both as seed files in migrations probs...